pages/linux/trash.md

```diff
@@ -22,3 +22,11 @@
 - Empty trash, keeping files trashed less than {{10}} days ago:
 
 `trash-empty {{10}}`
+
+- Remove all files matching name 'foo' from trash:
```

Comment:
In this case stuff like "barfoobar" would also be removed from trash? Or only "foo" exactly?

Reply:
Looks like exact name match from the source, since matching is done with fnmatch, unix wildcard style. I tested it to confirm.

https://github.com/andreafrancia/trash-cli/blob/5abecd53e1d84f2a5fd3fc60d2f5d71e518826c5/trashcli/rm.py#L84

Reply:
I'm not sure if it works for folders.